app/assets/javascripts/discourse/widgets/hamburger-categories.js.es6

```diff
@@ -5,7 +5,14 @@ import { number } from 'discourse/lib/formatter';
 createWidget('hamburger-category', {
   tagName: 'li.category-link',
 
+  buildAttributes(attrs) {
+    return { "data-category-url": Discourse.getURL("/c/") + Discourse.Category.slugFor(attrs) };
+  },
+
   html(c) {
+    if (c.parent_category_id)
```

Comment:
style wise I really much prefer 

if (x) { y }

I find omitting the {} quite often leads to weird and wonderful bugs